pkgs/top-level/all-packages.nix

```diff
@@ -637,6 +637,11 @@ with pkgs;
 
   genymotion = callPackage ../development/mobile/genymotion { };
 
+  gams = callPackage ../tools/misc/gams {
+    license = config.gams.license or null;
+    optgams = config.gams.optgams or null;
```

Comment:
I added a File suffix here for both, since it is not obvious what type is expected here.

Reply:
awesome